lib/beaker/dsl/install_utils.rb

```diff
@@ -826,7 +826,7 @@ def install_puppet_from_msi( host, opts )
           on host, "if not exist #{host['distmoduledir']} (md #{host['distmoduledir']})"
         end
 
-        on host, "msiexec /qn /i #{dest}"
+        on host, "cmd /C 'start /w msiexec.exe /qn /i #{dest}'"
```

Comment:
The previous command version was broken, curl on windows does not accept the second destination parameter.  So, we download the 'link' and it ends up in a destination 'dest', we use curl to download and msiexec to install.